amida-tech/hl7-telescoper-api-server
# Hl7 22 am list all parsed messages

### Related JIRA tickets:
-https://jira.amida.com/browse/HL7-22

### What exactly does this PR do?
- Added getParsedMessages function that retrieves all parsed messages for a single file given the fileId
- Eventually it should retrieve based on filename, but that needs to added in


### Do ANY of these changes introduce a breaking change? (in-scope or otherwise)
- No

### Manual test cases?
- Create a user, then login
- Upload a file with more than one message
- GET /api/hl7/files/parsedMessages/fileId and you should get a list of JSON objects with the same length as the number of messages in file


### Any additional context/background?
- N/A

### Screenshots (if appropriate):
